docs/sorting-algorithms/insertion-sort.mdx

```diff
@@ -0,0 +1,194 @@
+---
+title: Insertion Sort
+slug: /sorting-algorithm/insertion-sort
+---
+
+## Introduction
+
+A simple sorting algorithm which works on the idea of picking elements from the unsorted part one by one and inserting them into their right positions in the sorted part. The array is virtually divided into 2 parts, a sorted one and an unsorted one. Just like we arrange a group of shuffled pages according to their page numbers, values form unsorted part are picked and placed at the right positions in the sorted portion.
+
+![Imgur](https://i.imgur.com/Ora9Q1J.png)
+
```

Comment:
Is it possible to add a paragraph explaining the algorithm more intuitively? Maybe by sharing some example that the other articles do now?

Reply:
@HarshCasper is it fine if I explain the working of the example in the image?

Reply:
Totally fine. In fact it would be great :+1:

Reply:
Done.